pkg/ddc/alluxio/shutdown_test.go

```diff
@@ -268,10 +271,17 @@ func TestAlluxioEngineCleanAll(t *testing.T) {
 			testObjs := []runtime.Object{}
 			testObjs = append(testObjs, tt.fields.cm.DeepCopy())
 			client := fake.NewFakeClientWithScheme(testScheme, testObjs...)
+
```

Comment:
Do we need to add CleanUpFuse ut for Alluxio so we can make sure  it really follow : `common.LabelAnnotationFusePrefix + e.runtimeInfo.GetNamespace() + "-" + e.runtimeInfo.GetName()`

Reply:
CleanUpFuse ut include alluxio runtime type test, i think it is not necessary in alluxio shutdown test, WDYT.

Reply:
The original code is for jindofs. I'm not sure if it can work for Alluxio and goosefs. 

```
common.LabelAnnotationFusePrefix + e.runtimeInfo.GetNamespace() + "-" + e.runtimeInfo.GetName()
```

Reply:
Btw, did you test it manually?

Reply:
Yep, i have test yet.